src/ros_filter.cpp

```diff
@@ -557,6 +590,15 @@ namespace RobotLocalization
     // Determine if we're in 2D mode
     nhLocal_.param("two_d_mode", twoDMode_, false);
 
+    // Smoothing window size.
+    //
+    // Internally stored in ms because checking double values to be exactly zero is unreliable.
+    {
+      double filter_history_interval_seconds = 0.;
+      nhLocal_.param("filter_history_interval", filter_history_interval_seconds, 0.);
+      filterHistoryInterval_ = int(filter_history_interval_seconds*1000);
```

Comment:
In any case, if we must stick with this scheme of an integer and milliseconds internally, can we rename this variable to `filterHistoryIntervalMs_`?